### ISSUE TITLE
Can't reproduce the result of torch_vct

### ISSUE THREAD

**Issue by cccleon** [facebookresearch/NeuralCompression#209]
Thanks for your excellent implementation!

I tried to train VCT on 2 RTX 4090, a total of 100k step used. Initially I follow the training config mentioned in README but fail to reproduce.Then I adjust the learning rate and distortion lambda for several groups, PSNR/bpp is finally 35/0.11

Would you share the checkpoint?

**Comment by mmuckley** [comment]
Hello @cccleon, I'm going to CC @desi-ivanova for running the code while I check in on the status of releasing the checkpoint for you.

**Comment by desi-ivanova** [comment]
Hi @cccleon -- I don't think 2x4090s and 100K steps will get you there, unfortunately. The checkpoint @mmuckley is referring to was trained for 400K steps on 32 V100 (32GB) cards (as described in the readme).

**Comment by cccleon** [comment]
@desi-ivanova @mmuckley 
Thanks for your reply!
Looks like I should lower my learning rate and increase my total steps.I'll keep trying~
